src/edital/dto/create-etapas-edital.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { Type } from 'class-transformer';
+import { IsDate, IsNumber, IsOptional, IsString } from 'class-validator';
+
+export class CreateEtapasDto {
+  @IsString()
+  nome: string;
+
+  @IsString()
+  descricao: string;
+
+  @IsNumber()
+  ordem: number;
+
+  @IsDate()
+  @Type(() => Date)
+  data_inicio: Date;
+
+  @IsDate()
+  @Type(() => Date)
+  data_fim: Date;
+
+  @IsOptional()
+  @IsString()
+  descricao_etapa: string;
```

Comment:
Acredito que por estar com o Decorator de opcional, é boa prática adicionar o `?` pra indicar ao typescript que o atributo é opcional.

Obs: Necessário validar a mudança, pois não tenho certeza se isso pode afetar o funcionamento atual.

```suggestion
  descricao_etapa?: string;
```

Reply:
ja tava fazendo isso kkkkk